### ISSUE TITLE
Incorrect button style in Account Settings

### ISSUE THREAD

**Issue by daniellekirkwood** [element-hq/element-web#33410]
### Steps to reproduce

<img width="473" height="121" alt="Image" src="https://github.com/user-attachments/assets/77d68918-3ffd-4135-ba9e-bc018ceffac5" />

1. Where are you starting? What can you see?
I open settings and see this weird black and blue button

3. What do you click?
4. More steps…

### Outcome

<img width="184" height="96" alt="Image" src="https://github.com/user-attachments/assets/d50c2e4a-b1fd-4306-9fd9-37786e80eea3" />

<img width="267" height="93" alt="Image" src="https://github.com/user-attachments/assets/ad6b158a-2224-4804-bd13-470dfe175e00" />

I'm not sure which button it should be here, we should check the designs but either way it should have blue writing :) 

Element Nightly version: 2026050601
Crypto version: Rust SDK 0.16.0 (65b585f), Vodozemac 0.9.0

### Will you send logs?

Yes

**Comment by t3chguy** [comment]
Fixed by https://github.com/element-hq/element-web/pull/33411
